hiero-enterprise-base/src/main/java/com/openelements/hiero/base/IFileReader.java

```diff
@@ -0,0 +1,8 @@
+package com.openelements.hiero.base;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+public interface IFileReader {
```

Comment:
I disagree with this approach. Instead of mocking the Fiel read in the test you can easily create a test file and use that one in the tests